k6/README.md

```diff
@@ -0,0 +1,86 @@
+# k6 Integration
+
+## Overview
+
+Track performance metrics of [k6][1] tests to:
+- Correlate application performance with load testing metrics.
+- Create alerts based on performance testing metrics.
+- Analyze and visualize k6 metrics using the k6 Datadog Dashboard or [Metrics Explorer][5].
+
+![k6 Datadog Dashboard][9]
+
+## Setup
+
+For the detailed instructions, follow the [k6 documentation][2].
+
+### Installation
+
+1. In Datadog, navigate to Integrations > [API][3] to copy your API key.
+
+
+2. Run the Datadog Agent:
+
+To get k6 metrics into Datadog, k6 sends metrics through the Datadog Agent, which collects, aggregates, and forwards the metrics to the Datadog platform.
+
+Run the Datadog Agent service as a Docker container with this command:
+
+```shell
+DOCKER_CONTENT_TRUST=1 \
+docker run -d \
+    --name datadog \
+    -v /var/run/docker.sock:/var/run/docker.sock:ro \
+    -v /proc/:/host/proc/:ro \
+    -v /sys/fs/cgroup/:/host/sys/fs/cgroup:ro \
+    -e DD_SITE="datadoghq.com" \
+    -e DD_API_KEY=<YOUR_DATADOG_API_KEY> \
+    -e DD_DOGSTATSD_NON_LOCAL_TRAFFIC=1 \
+    -p 8125:8125/udp \
+    datadog/agent:latest
+```
+
+**Note**: Replace `<YOUR_DATADOG_API_KEY>` with your [API][3] key. If your account is registered with Datadog EU, change the value of `DD_SITE` to `datadoghq.eu`.
+
```

Comment:
Indent content below numbers 2, 3, and 4 by 4 spaces. This will make it display properly on the Datadog docs site.

Reply:
It should be fixed by https://github.com/DataDog/integrations-extras/pull/578/commits/7f7cda9f78aeb6e3258fdf2ad9d82f2c47d2fe8e. Let me know otherwise, thanks.